docs/dashboard.asciidoc

```diff
@@ -1,2 +1,117 @@
+[float]
 [[dashboard]]
-== Working with Dashboards
\ No newline at end of file
+== Dashboard
+
+A Kibana _dashboard_ displays saved visualizations in groups that you can arrange. You can save a dashboard to share or 
+reload at a later time.
+
+[float]
+[[getting-started]]
+=== Getting Started
+
+You need at least one saved <<visualize, visualization>> to use a dashboard.
+
+[float]
+[[creating-a-new-dashboard]]
+==== Creating a New Dashboard
+
+The first time you click the *Dashboard* tab, Kibana displays an empty dashboard.
+
+image:images/NewDashboard.png[New Dashboard screen]
+
+Add visualizations to start creating your dashboard.
```

Comment:
building your dashboard? An empty dashboard has already been created at this point.